pages/1.12/cli/command-reference/dcos-task/dcos-task-attach/index.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: layout.pug
+navigationTitle:  dcos task attach
+title: dcos task attach
+menuWeight: 0
+excerpt: Attaching a process inside of a task's container
+
+enterprise: false
+---
+
+# Description
+
+The `dcos task attach` command allows you to attach the CLI to the stdio of an already running task.
```

Comment:
I would add `To detach, type the sequence CTRL-p CTRL-q.`.

Reply:
done :)